.github/contributing.md

```diff
@@ -8,28 +8,22 @@ We try to keep both these branches in sync with each other the best we can. To d
 
 1. If you are working on a Formplayer change, you will want to start by checking out `your_feature_branch` from `formplayer` as the base branch. Make changes on `your_feature_branch` and create your original PR against `formplayer` branch.
 
-2. Now you will need to duplicate this PR by making another PR against `master`. Make sure the branch for this PR is not deleted. Then create the comment `duplicate this PR`. If the PR has already been merged, comment `duplicate this PR <starting-commit-id>`.
+2. Now you will need to duplicate this PR by making another PR against `master`. Make sure the branch for this PR is not deleted. Then create the comment `duplicate this PR`. If the PR has already been merged, comment `duplicate this PR <starting-commit-id>`. This should result in a Github Actions workflow duplicating your PR against `master`. Go to the duplicate PR, close and re-open it to run the Github checks against it. 
 
-3. In order for us to test that your PR against `master` doesn't break anything on CommCare Android, we need to run android side tests with your PR.
+3. In order for us to test that your PR against `master` doesn't break anything on CommCare Android, you need to run android side tests with your PR.
 To do this - 
-    - Check out a new branch say `test_cc_1189` from `master` in [CommCare Android](https://github.com/dimagi/commcare-android)
-    - Do an empty commit on your new branch as `git commit --allow-empty -m "Empty-Commit"` and push your branch to github. 
-    - Create a dummy test PR with a subject `[Test] CC 1189`. 
-    - You can remove all the fields from the PR template here and add a line at the very bottom saying `cross-request: link_to_commcare-core_master_pr`. Without this line the tests won't factor in your CommCare Core `master` PR to run tests with. 
-    - Label the PR with `cross requested tag`
-    - Label the PR with an appropriate `product/*` tag.
-    - You don't need to add any reviewers here as this PR will be closed without merging and is only used to run Android tests
+    - Go to the [Android test PR](https://github.com/dimagi/commcare-android/pull/2612)
+    - Change "cross-request" PR in description to the duplicate PR created in step 2 above
+    - Make a comment saying `@damagatchi retest this please` ([example](https://github.com/dimagi/commcare-android/pull/2612#issuecomment-1453132625))
```

Comment:
Is there an indicator if the test passed or fail? Would the PR owner need to anything if there's a failure?

Reply:
No, PR owner doesn't need to anything when there are failures. The build status should show up as pass or fail on the Android test PR.